examples/makenotebooks.py

```diff
@@ -105,8 +107,10 @@ def execute_notebook(fname):
         with open(fname, "w", encoding="utf-8") as f:
             nbformat.write(nb, f)
     except CellExecutionError:
-        raise Exception(f"Error executing the notebook {fname}")
+        print(f"ERROR executing {fname}")
+        return False
```

Comment:
I understand the print for the error is to avoid interrupting the execution, but is the return bool necessary? Seeing as there isn't any usage of that return within the makenotebook file.

Would it be better perhaps to have it this way?

```
except CellExecutionError:
    print(...)
    return # void return
print(...)
```

Reply:
I added that so that the calling context could determine whether the execution failed. It's true that the return value is not used at the moment, but is there any harm in leaving it in the function to support possible future changes in the calling code?

Reply:
Yeah definitely agree on the notation used, much better than straight up interrupting the exec. Also, I have no problems with leaving it in if its something you expect will be used in the future!